src/main/java/com/akademia/educators_back/entity/ProblemEntity.java

```diff
@@ -5,6 +5,9 @@
 
 import java.util.List;
 
+/**
+ * Entity of problem
```

Comment:
Problem był dał z dużej żeby zaakcentować co mamy na myśli, albo " Entity of Problem Object"